Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -122,9 +144,60 @@ def Druva_FindDevice_Command(clientObj, search_string):
     else:
         raise RuntimeError('Error: ' + str(response.status_code))
 
+def Druva_FindUser_Command(clientObj, user_string):
+    response = clientObj.get_findUser(user_string)
+    statusCode = response.status_code
+    if (statusCode==200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Found Druva users', responseJson['users'])
+        outputs = {"Druva.User(val.userID == obj.userID)": responseJson['users']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+
+    else:
+        raise RuntimeError('Error: ' + str(response.status_code))
```

Comment:
If there's an error in the request (`statusCode != 200`), does the endpoint provide an error message back (usually in `response.content` or `response.json()`)?

I think that providing the user with `Error: {statusCode}` doesn't contain enough information for the user to know what's wrong and how to fix their issue. Might be helpful to add the response content when you raise the `RuntimeError` in all `*_Command` methods.